project/FrontendBuild.scala

```diff
@@ -27,49 +27,42 @@ object FrontendBuild extends Build with MicroService {
 
 private object Versions {
 
-  val ficus         = "1.1.2"
-  val cacheClient   = "5.6.0"
-  val frontend      = "6.7.0"
-  val playConfig    = "2.1.0"
-  val playHealth    = "1.1.0"
-  val urlBuilder    = "1.0.0"
-  val httpclient    = "4.3.6"
-  val jsonLogger    = "2.1.1"
-  val scalatest     = "2.2.6"
-  val pegdown       = "1.6.0"
-  val jsoup         = "1.9.2"
-  val wiremock      = "1.57"
-  val hmrctest      = "1.8.0"
-  val scalatestplus = "1.2.0"
-  val silhouette    = "2.0.2"
-  val playWhitelist = "1.1.0"
+  val silhouette    = "4.0.0"
 }
 
 private object AppDependencies {
 
   import Versions._
 
   val compile = Seq(
-    "net.ceedubs"               %% "ficus"                    % ficus,
-    "uk.gov.hmrc"               %% "http-caching-client"      % cacheClient,
-    "uk.gov.hmrc"               %% "frontend-bootstrap"       % frontend,
-    "uk.gov.hmrc"               %% "play-config"              % playConfig,
-    "uk.gov.hmrc"               %% "play-json-logger"         % jsonLogger,
-    "uk.gov.hmrc"               %% "play-health"              % playHealth,
-    "uk.gov.hmrc"               %% "url-builder"              % urlBuilder,
-    "uk.gov.hmrc"               %% "play-whitelist-filter"    % playWhitelist,
-    "org.apache.httpcomponents" %  "httpclient"               % httpclient,
-    "com.mohiva"                %% "play-silhouette"          % silhouette
+    "com.iheart"                %% "ficus"                                    % "1.2.6",
+    "uk.gov.hmrc"               %% "http-caching-client"                      % "6.1.0",
+    "uk.gov.hmrc"               %% "frontend-bootstrap"                       % "7.10.0",
+    "uk.gov.hmrc"               %% "play-config"                              % "3.1.0",
```

Comment:
It was defensive against the session timeout filter problem, but i think 7.14 is the first version to have the fix, we weren't sure what was causing it hence the reason config was also downgraded. I've upgraded it and am just rerunning acceptance.